codebase/config/sync/core.entity_view_display.node.islandora_object.open_seadragon.yml

```diff
@@ -215,17 +225,37 @@ content:
     settings:
       link: true
     third_party_settings: {  }
+  field_dspace_identifier:
```

Comment:
Do you know why the OpenSeadragon view (if that's the right word) modifies a bunch of field information unrelated to the custodial history?  E.g. dspace id, jhir?

Reply:
This is just a display mode and I realized when I was in there that these other fields weren't available through that display mode.  I'm not sure if the UI is using these modes. 

I'll take these out.